docs/index.html

```diff
@@ -58,6 +58,8 @@
 
 // define access token
 mapboxgl.accessToken = 'pk.eyJ1IjoibWlrZWxtYXJvbiIsImEiOiJjaWZlY25lZGQ2cTJjc2trbmdiZDdjYjllIn0.Wx1n0X7aeCQyDTnK6_mrGw';
+// cdn base address
+const cdnBase = 'https://cdn.jsdelivr.net/gh/osmlab/osm-community-index@main/dist/';
```

Comment:
Need to stick to `var` to keep things working in IE11 without introducing a build step.